src/web/app/pages-instructor/instructor-courses-page/__snapshots__/instructor-courses-page.component.spec.ts.snap

```diff
@@ -813,163 +825,167 @@ exports[`InstructorCoursesPageComponent should snap when new course form is expa
           </button>
-        </div>
-        <div
-          class="row"
-        >
-          <div
-            class="col-md-6"
-          >
-            <div
-              class="row align-items-center"
-            >
-              <div
-                class="col-md-6"
-              >
-                <h5
-                  class="text-center"
-                >
-                  Create new Course
-                </h5>
-              </div>
-            </div>
-          </div>
-          <div
-            class="col-md-6"
+          <button
+            class="btn btn-primary btn-sm"
+            id="btn-close-form"
+            style=""
           >
-            <div
-              class="row text-center align-items-center mt-1"
-            >
-              <div
-                class="col-md-3"
-              >
-                <h5
-                  class="text-md-right"
-                >
-                  Or: 
-                </h5>
-              </div>
-              <div
-                class="col-md-9 text-md-left"
-              >
-                <button
-                  class="btn btn-info"
-                  disabled=""
-                  id="btn-copy-course"
-                  type="button"
-                >
-                   Copy from previous courses 
-                </button>
-              </div>
-            </div>
-          </div>
+            <i
+              aria-hidden="true"
+              class="fa fa-times"
+            />
+          </button>
         </div>
-        <div
-          class="card-body text-center"
+      </div>
+      <div
+        class="card-body fill-plain"
+      >
+        <form
+          class="ng-untouched ng-pristine ng-valid"
+          novalidate=""
         >
           <div
-            class="form-group row"
+            class="form-row form-group"
           >
             <label
-              class="col-md-3 text-md-right col-form-label"
+              class="col-sm-2 font-weight-bold text-md-right"
             >
               Course ID:
             </label>
             <div
-              class="col-md-6 text-md-left"
+              class="col-sm-9"
             >
               <input
-                class="form-control ng-untouched ng-pristine ng-valid"
-                id="new-course-id"
+                class="col-xs-12 form-control ng-untouched ng-pristine ng-valid"
+                id="course-id"
                 maxlength="64"
+                name="courseId"
                 placeholder="e.g. CS3215-2013Semester1"
+                required=""
+                tabindex="1"
                 type="text"
               />
+              <div
+                class="invalid-field"
+                hidden=""
+              >
+                <i
+                  aria-hidden="true"
+                  class="fa fa-exclamation-circle"
+                />
+                 The field Course ID should not be empty. 
+              </div>
```

Comment:
Should the snapshot here change to display the validation error?

Reply:
Will look into this, thanks for pointing it out!

Reply:
@FergusMok  I investigated into this, the boolean computation used for hidden directive, it seems to be correct. The invalid field message does not show when viewing the component using the browser. Furthermore, if we resort to work with `ngIf` directive instead `hidden`, it works as desired both in snapshot and browser. Shall I proceed to use `ngIf`?

Reply:
@thikhinab 
After reading more https://stackoverflow.com/a/51336717, I realize the `hidden=""` and `aria-hidden="true"` here means that the validation error is actually not shown but rendered, which is actually the correct behavior. Not entirely sure why the previous implementation did not have it. I think sticking to hidden for now should be correct too. Thanks for looking into this 🙏

Reply:
@FergusMok Thanks for sharing this! It paints a better picture of the scenario.

Reply:
@thikhinab Remember to revert the change! 👍